packages/core/ui-settings/core-ui-settings-server-internal/src/settings/theme.ts

```diff
@@ -65,6 +65,12 @@ export const getThemeSettings = (
           defaultMessage: `Sync with system`,
         }),
       },
+      deprecation: {
+        message: i18n.translate('core.ui_settings.params.darkModeDeprecation', {
+          defaultMessage: 'This setting is deprecated and will be removed in Kibana 10.0.',
```

Comment:
Should we mention the alternative here?

Reply:
The problem is that, depending on where Kibana runs, the way to set the theme is either through user profile, either through the user dropdown new "appearance" menu. If you have a good text copy suggestion I am all ear 😊

Reply:
got it :(